### ISSUE TITLE
clean up builddir mess

### ISSUE THREAD

**Issue by thvitt** [faustedition/faust-gen-html#646]
for historical reasons, there are various options conveying output directories. The main method is the config.xml file which is read by the xproc pipelines and partly passed on as parameters to the individual stylesheets. The stylesheets sometimes have fallback values that try to guess a path. The config file exists in a 'relative path' version that is checked in and in an 'absolute path' version that is created by the maven script.

Basically, with maximum support, we have the following main paths that can be flexible:
* source XML files
* builddir (with convention-based layout of subdirectories)

and we have the following possible use cases:
* run from aggregation build, i.e. faust-gen project, global source and target folders
* run from xproc build, no faust-gen project
* run individual pipeline
* run individual xslt
